atd-vze/src/views/Locations/LocationNotes.js

```diff
@@ -82,10 +82,8 @@ const LocationNotes = ({ locationId }) => {
       },
     })
       .then(response => {
-        refetch().then(response => {
-          setEditedNote("");
-          setEditRow("");
-        });
+        refetch();
```

Comment:
I'll test out Location notes, but I suspect that Rose was making a small fix here. 👍